src/java/com/github/jsdossier/CompilerModule.java

```diff
@@ -78,7 +79,7 @@ CompilerOptions provideCompilerOptions(
 
     // IDE mode must be enabled or all of the jsdoc info will be stripped from the AST.
     options.setIdeMode(true);
-    options.setPreserveJsDocWhitespace(true);
+    options.setParseJsDocDocumentation(Config.JsDocParsing.INCLUDE_DESCRIPTIONS_WITH_WHITESPACE);
```

Comment:
> Guess I could go back to a snapshot (e522685) - or are there compiler snapshots on sonatype (already have a snapshot dep for guava)?

I don't think an older snapshot will work. This is broken because we're building in the head compiler internally. Was there a particular reason why this code isn't in Piper?

> On a more general note, does the compiler have a deprecation policy for public APIs? I could see this causing headaches for anyone else building extensions for the compiler

Use of the Java API externally is quite rare. AFAIK, there are no formal policies around backwards incompatible changes for open-source users. However, per Ben, internal API usage usually requires backwards incompatible API changes to be more gradual, giving external users an early warning something is broken.